Directory.Packages.props

```diff
@@ -109,7 +109,7 @@
     <PackageVersion Include="Npgsql.DependencyInjection" Version="8.0.2" />
     <PackageVersion Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="8.0.2" />
     <PackageVersion Include="Oracle.EntityFrameworkCore" Version="8.23.40" />
-    <PackageVersion Include="Polly.Core" Version="8.3.1" />
+    <PackageVersion Include="Polly.Core" Version="8.4.0" />
     <PackageVersion Include="Polly.Extensions" Version="8.3.1" />
```

Comment:
Why didn't `Polly.Extensions` get bumped to 8.4.0? https://www.nuget.org/packages/Polly.Extensions